linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_unlock_time/ansible/shared.yml

```diff
@@ -0,0 +1,50 @@
+# platform = multi_platform_sle,multi_platform_ubuntu
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+{{{ ansible_instantiate_variables("var_accounts_passwords_pam_tally2_unlock_time") }}}
+
+- name: Check to see if pam_tally2.so is configured in /etc/pam.d/common-auth
```

Comment:
You can use the Ansible macros for PAM settings: `ansible_ensure_pam_module_option`
This would make this remediation much simpler and also aligned with other PAM rules.